src/System.Security.Cryptography.Encoding/tests/project.json

```diff
@@ -1,10 +1,11 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-rc4-24213-03",
-    "System.Linq": "4.0.0",
+    "System.Linq": "4.1.0-rc4-24213-03",
     "System.Linq.Expressions": "4.1.0-rc4-24213-03",
     "System.ObjectModel": "4.0.12-rc4-24213-03",
     "System.Runtime": "4.1.0-rc4-24213-03",
+    "System.Resources.ResourceManager": "4.0.1-rc4-24213-03",
```

Comment:
Previous incarnation of these changes failed due to no longer resolving System.Resources.ResourceManager during build.